server/tests/services/group.test.js

```diff
@@ -0,0 +1,140 @@
+import httpStatus from 'http-status';
+import groupService from '../../services/group';
+import Group from '../../models/group';
+
+describe('# Group Service', () => {
+  const groupObject = {
+    name: 'Wondertwins',
+    users: ['123456789012345678901234']
+  };
+
+  context('with no groups in the database beforehand', () => {
+    beforeEach(() => {
+      Group.remove({}, () => {});
+    });
+
+    describe('createGroup()', () => {
+      context('with valid group details', () => {
+        it('creates a new group', (done) => {
+          groupService.createGroup(groupObject)
+            .then((createdGroup) => {
+              expect(createdGroup).to.exist;
+              expect(createdGroup.name).to.equal(groupObject.name);
+              expect(createdGroup.users[0].toString()).to.equal(groupObject.users[0]);
+              done();
+            })
+            .catch(err => done(err));
+        });
+      });
+
+      context('with missing user details', () => {
+        const missingName = {
```

Comment:
nit: can you add specifics here? `missingNameGroup` just so you can know what it is without looking around